content/actions/security-guides/using-secrets-in-github-actions.md

```diff
@@ -403,7 +403,7 @@ You can use Base64 encoding to store small binary blobs as secrets. You can then
 1. Use `base64` to encode your file into a Base64 string. For example:
 
```

Comment:
I think macos doesn't support the -w option (https://www.unix.com/man-page/osx/1/base64/), so maybe we can use both examples, and specify the platform:

````suggestion

On MacOS, you could run:

   ```shell
      base64 -i cert.der -o cert.base64
   ```

On Linux, you could run:

````